firmware/gameport-adapter/CH-f16-combat-stick.h

```diff
@@ -0,0 +1,88 @@
+// This file is part of Necroware's GamePort adapter firmware.
+// Copyright (C) 2021 Necroware
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+//
+// CH F16 Combat Stick from 1995
+// Made by Stefan Geissler
```

Comment:
No need to mention your name in the file. It can and will be modified by another users one day and git remembers which line has been changed by which user, so your name is already included. Check it out using `git blame` :)